Дубровин И.В. Лаб. 1 Группа 6311

ctwoth opened the pull request:
ФИО: Дубровин Илья Викторович
Номер группы: 6311
Номер лабораторной: 1
Краткое описание исходного кода: взял свой код из предыдущего семестра по предмету Основы Информационной безопасности
Краткое описание проделанной работы: Исправил код в соответствии со стандартами кодирования (отступы, названия классов, сложные индексные выражения, комментарии, докстринги и т.д)

ctwoth committed:
add bad unreadable code

ctwoth committed:
fix margins to from 2 to 4 space

ctwoth committed:
class names allways CamleCase

ctwoth committed:
added spaces between classes, functions, and methods

ctwoth committed:
comments should be at the code level.

ctwoth committed:
fix long strings, add space in complex index expression and funk calls

ctwoth committed:
Declarate funk params type and return type

ctwoth committed:
add docstrings and some small fixes

IlyaOv reviewed (changes requested):
Если приводили к PEP8, то
- не везде длина строк исправлена
- лишние пробелы в строках

ctwoth committed:
fix main.py

убрал лишние пробелы.
исправил большие строки при помощи переносов или разделения сложные выражений на несколько простых.

IlyaOv reviewed (approved):
Зачтено
